<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QDialog" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="lwText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
   <property name="title">
    <string>Player 1</string>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lUsername">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
   <property name="title">
    <string>Player 2</string>
   </property>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lUsername_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
   <property name="title">
    <string>Player 3</string>
   </property>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lUsername_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
